packages/reputation-miner/test/MaliciousReputationMinerWrongResponse.js

```diff
@@ -62,62 +62,55 @@ class MaliciousReputationMinerWrongResponse extends ReputationMinerTestWrapper {
+        this.responseToFalsify === 21 ? this.responseValue : lastAgreeJustifications.adjacentReputationProof.reputation,
+        this.responseToFalsify === 22 ? this.responseValue : lastAgreeJustifications.adjacentReputationProof.uid,
+        this.responseToFalsify === 23 ? this.responseValue : "0",
+        this.responseToFalsify === 24 ? this.responseValue : lastAgreeJustifications.originAdjacentReputationProof.reputation,
+        this.responseToFalsify === 25 ? this.responseValue : lastAgreeJustifications.childAdjacentReputationProof.reputation
       ],
       [
-        this.responseToFalsify === 29 ?
+        this.responseToFalsify === 26 ?
           this.responseValue :
           ethers.utils.hexZeroPad(ReputationMinerTestWrapper.breakKeyInToElements(reputationKey)[0], 32),
-        this.responseToFalsify === 30 ?
+        this.responseToFalsify === 27 ?
           this.responseValue :
           ethers.utils.hexZeroPad(ReputationMinerTestWrapper.breakKeyInToElements(reputationKey)[1], 32),
-        this.responseToFalsify === 31 ?
+        this.responseToFalsify === 28 ?
           this.responseValue :
           ethers.utils.hexZeroPad(ReputationMinerTestWrapper.breakKeyInToElements(reputationKey)[2], 32),
-        this.responseToFalsify === 32 ?
+        this.responseToFalsify === 29 ?
           this.responseValue :
             soliditySha3(reputationKey),
-        this.responseToFalsify === 33 ?
-          this.responseValue :
-            soliditySha3(lastAgreeJustifications.newestReputationProof.key),
-        this.responseToFalsify === 34 ?
+        this.responseToFalsify === 30 ?
           this.responseValue :
             soliditySha3(lastAgreeJustifications.adjacentReputationProof.key),
-        this.responseToFalsify === 35 ?
+        this.responseToFalsify === 31 ?
           this.responseValue :
           soliditySha3(lastAgreeJustifications.originAdjacentReputationProof.key),
-        this.responseToFalsify === 36 ?
+        this.responseToFalsify === 32 ?
           this.responseValue :
           soliditySha3(lastAgreeJustifications.childAdjacentReputationProof.key)
       ],
-      this.responseToFalsify === 37 ? this.responseValue : firstDisagreeJustifications.justUpdatedProof.siblings,
-      this.responseToFalsify === 38 ? this.responseValue : agreeStateSiblings,
-      this.responseToFalsify === 39 ? this.responseValue : disagreeStateSiblings,
-      this.responseToFalsify === 40 ? this.responseValue : lastAgreeJustifications.newestReputationProof.siblings,
-      this.responseToFalsify === 41 ? this.responseValue : lastAgreeJustifications.originReputationProof.siblings,
-      this.responseToFalsify === 42 ? this.responseValue : lastAgreeJustifications.childReputationProof.siblings,
-      this.responseToFalsify === 43 ? this.responseValue : lastAgreeJustifications.adjacentReputationProof.siblings,
```

Comment:
My one question is -- if there are a total of 4 arguments removed (3 + 1), why has the number of options here been reduced by 5?

Reply:
By 3+1, I assume you mean `uint256[29] -> uint256[26]` and `bytes32[8] -> bytes32[7]` but the array of siblings for the relevant proof [is also removed](https://github.com/JoinColony/colonyNetwork/pull/731/files/22f2f4c47b3f69dd40667ae717bfebbcd97f5e37..23aa9d62470e8596cf8515369080be3afa95bdfa#diff-38a06e49a953fabbbea435027d827918L157)